src/removeNPMAbsolutePaths.js

```diff
@@ -36,20 +36,26 @@ function processFile(filePath, opts) {
   })
     .then((data) => {
       let writeFile = false;
+      let errorFound = false;
       let obj;
       try {
         obj = JSON.parse(data);
       } catch (err) {
-        throw createError(`Malformed package.json file at "${filePath}"`, err);
```

Comment:
This throw was find since it's caught on the next `.then` second callback and the library just continues.